Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Datum.h

```diff
@@ -152,15 +152,15 @@ namespace ScriptCanvas
         void SetNotificationsTarget(AZ::EntityId notificationId);
 
         // pushes this datum to the void* address in destination
-        bool ToBehaviorContext(AZ::BehaviorValueParameter& destination) const;
+        bool ToBehaviorContext(AZ::BehaviorArgument& destination) const;
 
         // returns BVP with void* to the datum address
-        AZ::BehaviorValueParameter ToBehaviorContext(AZ::BehaviorClass*& behaviorClass);
+        AZ::BehaviorArgument ToBehaviorContext(AZ::BehaviorClass*& behaviorClass);
 
-        // creates an AZ::BehaviorValueParameter with a void* that points to this datum, depending on what the parameter needs
-        // this is called when the AZ::BehaviorValueParameter needs this value as input to another function
+        // creates an AZ::BehaviorArgument with a void* that points to this datum, depending on what the parameter needs
+        // this is called when the AZ::BehaviorArgument needs this value as input to another function
         // so it is appropriate for the value output to be nullptr
-        AZ::Outcome<AZ::BehaviorValueParameter, AZStd::string> ToBehaviorValueParameter(const AZ::BehaviorParameter& description) const;
+        AZ::Outcome<AZ::BehaviorArgument, AZStd::string> ToBehaviorValueParameter(const AZ::BehaviorParameter& description) const;
```

Comment:
This function should be renamed now as it is using the supply BehaviorParameter instance to make a BehaviorArgument instance using the value stored in the datum